nexus/src/app/background/tasks/sync_switch_configuration.rs

```diff
@@ -71,6 +71,10 @@ const PHY0: &str = "phy0";
 // before breaking to check for shutdown conditions.
 const BGP_SESSION_RESOLUTION: u64 = 100;
 
+// This is the default RIB Priority used for static routes.  This mirrors
+// the const defined in maghemite in rdb/src/lib.rs.
+const DEFAULT_RIB_PRIORITY_STATIC: u8 = 1;
```

Comment:
Should this be imported from the rdb lib (where it's pub)?

Reply:
Yeah, definitely.  I'll address this in a follow-up PR, so we can get this merged + omicron/maghemite back in sync for R11.